Mathlib/Algebra/Star/Conjneg.lean

```diff
@@ -0,0 +1,78 @@
+/-
+Copyright (c) 2024 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import Mathlib.Algebra.BigOperators.Pi
+import Mathlib.Algebra.Star.Pi
+import Mathlib.Data.Set.Pointwise.Basic
+
+/-!
+# Conjugation-negation operator
+
+This file defines the conjugation-negation operator, useful in Fourier analysis.
+
+The way this operator enters the picture is that the adjoint of convolution with a function `f` is
+convolution with `conjneg f`.
+-/
+
+open Function
+open scoped ComplexConjugate
+
+variable {ι G R : Type*} [AddGroup G]
+
+section CommSemiring
+variable [CommSemiring R] [StarRing R] {f g : G → R}
+
+/-- Conjugation-negation. Sends `f` to `fun x ↦ conj (f (-x))`. -/
+def conjneg (f : G → R) : G → R := conj fun x ↦ f (-x)
```

Comment:
Does it make sense to make this a bundled ring hom?

Reply:
I don't like making functions bundled by default because it makes them always be prepended with `⇑`, have their API dispersed across the general hom APIs and their own, and is generally little to not useful. I have instead added a bundled version `conjnegRingHom`